scripts/stac/imagery/collection.py

```diff
@@ -75,6 +83,41 @@ def __init__(
 
         self.add_providers(providers)
 
+    def add_capture_area(self, polygons: List[shapely.geometry.shape], target: str) -> None:
+        """Add the capture area of the Collection.
+        The `href` or path of the capture-area.geojson is always set as the relative `./capture-area.geojson`
+
+        Args:
+            polygons: list of geometries
+            target: path of the capture-area-geojson file
+        """
+
+        # The GSD is currently alway in meters (e.g. `0.3m`)
```

Comment:
Why "currently"? Is it going to be a different unit in the future? If not, we should probably avoid that phrasing

Also, typo, punctuation after "e.g.". Suggestion:

```suggestion
        # The GSD is measured in meters (e.g., `0.3m`)
```